payload/builder.go

```diff
@@ -324,6 +326,58 @@ func (p *Payload) SoundVolume(volume float32) *Payload {
 	return p
 }
 
+// InterruptionLevelPassive sets the aps interruption-level to "passive" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":passive}}
+func (p *Payload) InterruptionLevelPassive() *Payload {
+	p.aps().InterruptionLevel = "passive"
+	return p
+}
+
+// InterruptionLevelActive sets the aps interruption-level to "active" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":active}}
+func (p *Payload) InterruptionLevelActive() *Payload {
+	p.aps().InterruptionLevel = "active"
+	return p
+}
+
+// InterruptionLevelTimeSensitive sets the aps interruption-level to "time-sensitive" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":time-sensitive}}
+func (p *Payload) InterruptionLevelTimeSensitive() *Payload {
+	p.aps().InterruptionLevel = "time-sensitive"
+	return p
+}
+
+// InterruptionLevelCritical sets the aps interruption-level to "critical" on the payload.
+// This is to indicate the importance and delivery timing of a notification.
+// This interruption level requires an approved entitlement from Apple.
+// See: https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/
+//
+// {"aps":{"interruption-level":critical}}
+func (p *Payload) InterruptionLevelCritical() *Payload {
+	p.aps().InterruptionLevel = "critical"
+	return p
+}
+
+// The relevance score, a number between 0 and 1,
+// that the system uses to sort the notifications from your app.
+// The highest score gets featured in the notification summary.
+// See https://developer.apple.com/documentation/usernotifications/unnotificationcontent/3821031-relevancescore.
+//
+//	{"aps":{"relevance-score":0.1}}
+func (p *Payload) RelevanceScore(b float32) *Payload {
+	p.aps().RelevanceScore = b
+	return p
+}
```

Comment:
This doesn't appear to allow setting a `relevance-score` of `0` because it will omitted when building the `aps` JSON. This will result in  I think you can address this by either using the `Badge` precedent above (preferred) or by changing the type to a pointer.

Reply:
I can make a PR for this if you'd like.

Reply:
Good catch @chrishaines on relevance-score. 

Sure if you have time to do the PR for that, and I will try and get the other swapped out tomorrow or at the weekend.

Did you see my question re the `E` prefix above?

Thanks.

Reply:
Here ya go!
https://github.com/neilmorton/apns2/pull/1

Reply:
Thank you!